ignite/distributed/comp_models/native.py

```diff
@@ -371,8 +371,14 @@ def spawn(  # type: ignore[override]
 
             start_processes = mp.spawn
             # start_method and start_processes in pytorch >= 1.5
+            default_start_method = "spawn"
```

Comment:
I’m not sure but why not 
```Python
# start_method and start_processes in pytorch >= 1.5
if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
    If '__IPYTHON__' in globals():
        # use fork in jupyter
        spawn_kwargs["start_method"] = "fork"
    else:
        # let the user choose or spawn 
        spawn_kwargs["start_method"] = kwargs.get("start_method", "spawn")
        start_processes = mp.start_processes
```

@Chandan-h-509 what do you think ?

Reply:
Let's override default value to let user the control if he/she defines start method

Reply:
> I’m not sure but why not
> 
> ```python
> # start_method and start_processes in pytorch >= 1.5
> if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
>     If '__IPYTHON__' in globals():
>         # use fork in jupyter
>         start_method = "fork"
>     else:
>         # let the user choose or spawn 
>         start_method = kwargs.get("start_method", "spawn")
>     spawn_kwargs["start_method"] = start_method
>     start_processes = mp.start_processes
> ```
> 
> @Chandan-h-509 what do you think ?

I am fine with it.
What do you think @vfdev-5

Reply:
> What do you think @vfdev-5

Please, see https://github.com/pytorch/ignite/pull/2188#discussion_r707983971